syz-ci: smoke test panic in case of fuzzing_vms > 0

If `fuzzing_vms` > 0, we get the following panic during the smoke test

```
panic: makechan: size out of range

goroutine 1 [running]:
main.newReproManager({0x20a63f0?, 0xc001a8ed00}, 0xffffffffffffffdb, 0x0)
        /syzkaller/gopath/src/github.com/google/syzkaller/syz-manager/repro.go:46 +0x69
main.RunManager(0x1, 0xc001f62900)
        /syzkaller/gopath/src/github.com/google/syzkaller/syz-manager/manager.go:315 +0x992
main.main()
        /syzkaller/gopath/src/github.com/google/syzkaller/syz-manager/manager.go:214 +0x406
```

We override the number of VMs here:
https://github.com/google/syzkaller/blob/bde81f6f54f0b2fa4440884eeffa7db36b70f837/syz-ci/manager.go#L561-L563

I think we should also patch `fuzzing_vms` in `instance.OverrideVMCount()`.

--- a/pkg/instance/instance.go
+++ b/pkg/instance/instance.go
@@ -212,6 +212,7 @@ func OverrideVMCount(cfg *mgrconfig.Config, n int) error {
 		return fmt.Errorf("failed to serialize VM config: %w", err)
 	}
 	cfg.VM = vmCfg
+	cfg.FuzzingVMs = min(cfg.FuzzingVMs, n)
 	return nil
 }
 

